scripts/rollup/build.js

```diff
@@ -540,7 +544,7 @@ async function createBundle(bundle, bundleType) {
   const rollupConfig = {
     input: resolvedEntry,
     treeshake: {
-      pureExternalModules,
+      moduleSideEffects: pureExternalModules,
```

Comment:
- https://github.com/rollup/rollup/blob/master/CHANGELOG.md#200
  - > The `treeshake.pureExternalModules` option will now show a deprecation warning when used: use `treeshake.moduleSideEffects: 'no-external'` instead
- https://rollupjs.org/guide/en/#treeshakepureexternalmodules